src/analysis/retail/priest/holy/modules/talents/BottomRow/EmpoweredRenew.tsx

```diff
@@ -31,25 +30,13 @@ class EmpoweredRenew extends Analyzer {
   get effectiveHealing() {
     return (
       this.abilityTracker.getAbility(SPELLS.EMPOWERED_RENEW_TALENT_HEAL.id).healingEffective +
-      this.abilityTracker.getAbility(SPELLS.EMPOWERED_RENEW_TALENT_HEAL.id).healingAbsorbed +
-      this.echoOfLightMastery.masteryHealingBySpell[SPELLS.EMPOWERED_RENEW_TALENT_HEAL.id]
```

Comment:
removing mastery from this since we are not including mastery in any other has modules. also was causing bug if crash if player never casted renew with this talent